.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        ruby: ["2.7", "3.0", "3.1", "3.2"]
+        ruby: ["2.7", "3.0", "3.1", "3.2", "3.3"]
```

Comment:
Alternatively, we could limit `macos-latest` and `window-latest` to run only against 3.2.

Reply:
I would probably advocate to keep the matrix since it could uncover an issue for a specific Ruby version that we wouldn't be able to catch otherwise.

Ruby 2.7 is already EOL, so hopefully we can remove it from the matrix soon as update our requirement for > 3.0. We usually give users a buffer, so maybe we can remove it once 3.3 is stable.